services/user-svc/internal/organization/organization.go

```diff
@@ -0,0 +1,193 @@
+package organization
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	pb "gen/proto/services/user_svc/v1"
+	"github.com/google/uuid"
+	"github.com/rs/zerolog"
+	zlog "github.com/rs/zerolog/log"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"gorm.io/gorm"
+	"hwgorm"
+)
+
+type Base struct {
```

Comment:
Please use eloquent variable names

Reply:
As discussed: An eloquent name would be `OrganizationBase` but the golinter reduced this down to `Base` because the package name is already `organization`.